ui/src/jobs/components/JobStatusAlert.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Row, Col, Alert } from 'antd';
+import { JOB_STATUSES } from '../../common/constants';
+
+function JobStatusAlert({ status }) {
+  const types = { pending: 'warning', closed: 'error' };
+
+  return (
+    (status === 'pending' || status === 'closed') && (
+      <Row className="mb2">
+        <Col>
```

Comment:
`Row` and `Col` is the part of detail page layout, they should stay there, you can add `mb2` to `Alert` directly.

Reply:
Shouldn't the `mb2` stay in the detail page `Row` then?